ProjectOutline.md

```diff
@@ -0,0 +1,36 @@
+#App Function
+User searches a start city (optional) and a destination city,
+and the UI displays route data, weather data, and things to do in the destination city.
+Integration of historical user searches is also planned.
+
+#UX
+User Log in (Email/PW)
+User Register (if not already registered, New Panel for Registration)
+
+Previous City Searches, Trending City visits, Seasonal Recomendations
+
+Main Search Panel with Search form and submit buttons
+
+Results panel
+Map,
+Weather (10 day forecast, + possible historic data),
+Travel Route,
+Things to do,
+hotels,
+flights
```

Comment:
We might take out the map, hotels, and flights (for now). If we get a functional "base app" working then we can look at adding in more content like that.

Reply:
We can change later.